packages/camera/camera_avfoundation/ios/Classes/CameraProperties.m

```diff
@@ -104,9 +104,9 @@ UIDeviceOrientation FLTGetUIDeviceOrientationForString(NSString *orientation) {
   switch (orientation) {
     case UIDeviceOrientationPortraitUpsideDown:
       return @"portraitDown";
-    case UIDeviceOrientationLandscapeRight:
-      return @"landscapeLeft";
```

Comment:
Could you dig into the history and see why we had this flipped before? was it just a mistake?

Reply:
Yes. This should be a mistake when learning the relation between `AVCaptureVideoOrientation`, `UIInterfaceOrientation`, and `UIDeviceOrientation`.

`AVCaptureVideoOrientation` should be flipped with the corresponding `UIDeviceOrientation` by applying the affine transform, which typically means:
- `AVCaptureVideoOrientationLandscapeLeft` <=> `UIDeviceOrientationLandscapeRight`
- `AVCaptureVideoOrientationLandscapeRight` <=> `UIDeviceOrientationLandscapeLeft`

But the previous PR over-handled the implementation by replacing those string results too, which makes it a mistake. The value corresponding to orientation strings should always be as-is.

*See also: https://stackoverflow.com/questions/7845520/why-does-avcapturevideoorientation-landscape-modes-result-in-upside-down-still-i*